tools/mpidbg.sh

```
@@ -0,0 +1,31 @@
+
+(
+  # do LD_PRELOAD HERE if required for ASAN
+
+  ## for debugging mpirun in parallel processes
+  # gdbserver localhost:$PORT $@
+
+  ## for sampling function times
+  # perf record -o "perf.${RANK}.data" -Tga -F 1000 $@
+
+  ## stop/print on first error
+  gdb -batch -ex run -ex bt --args $@
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)